_sub/monitoring/helm-exporter/vars.tf

```diff
@@ -38,7 +38,7 @@ variable "repo_branch" {
 
 variable "helm_chart_version" {
   type        = string
-  description = "The version of the helm Exporter Helm Chart that should be used"
+  description = "The version of the Helm Exporter Helm Chart that should be used"
   default     = null
 }
 
```

Comment:
Looks like you are missing a variable here called target_namespaces. Then in the dependencies.tf file, you will need to assign target_namespaces to the Helm chart values, for instance on a new line below this line: https://github.com/dfds/infrastructure-modules/blob/master/_sub/monitoring/helm-exporter/dependencies.tf#L63

The helm chart vale for this is simply called namespaces, and is a comma separated string.


https://github.com/sstarcher/helm-exporter/blob/master/helm/values.yaml#L2

Reply:
Ok i have done what i think you have suggested here please check